awscli/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apk add --no-cache \
         less>=590-r0 \
         python3~3.10 \
         py3-pip~22 \
+        openssh \
```

Comment:
I know you are just moving things from paas-bootstrap. However this feels like it should be 'openssh-client'. I don't think we need the server packages.